.vscode/defsettings.json

```diff
@@ -5,6 +5,7 @@
     "deckkey"   : "-i ${env:HOME}/.ssh/id_rsa",
     "deckdir"   : "/home/deck",
     "python.analysis.extraPaths": [
+        "../decky-loader/plugin",
```

Comment:
We shouldn't expect plugin developers to have a local clone of the loader. Is there any other way this can be done?

Reply:
They could also just download the single `decky_plugin.py` to their `py_modules` folder and they are good to go :)

Reply:
IMO that should not be done so the plugin.py isn't copied into the plugin zip

maybe making it download a pyi would make more sense

Reply:
Ohh, `.pyi` is a python stub file right ? Yeah that's a good idea, I'll try that. So I simply write the signatures or constants followed by 3 dots `...` right ?

Where do you think it would make most sense to keep it ? In a dedicated `plugin/` folder here to mirror the folder in the decky-loader, in `py_modules` or alongside `main.py` (the idea was that maybe in the future this public interface grows with more files so putting it alongside might clutter things up) or a script that would download it from the decky-loader repo (which would mean the `.pyi` would be stored in the decky-loader repo ?) ?

Reply:
Ok, so I included a versioned `.pyi` of `decky_plugin.py` alongside `main.py` for now. We can always make things more complex when the time comes to do so.